bbi_summary_test.go

```diff
@@ -0,0 +1,63 @@
+package babylontest
+
+import (
+	"context"
+	"github.com/stretchr/testify/require"
+	"path/filepath"
+	"testing"
+)
+
+type testConfig struct {
+	bbiFlag string
+	goldenExt string
+}
+
+var testConfigs = []testConfig{
+	testConfig{
+		"--json",
+		".json",
+	},
+	testConfig{
+		"",
+		".txt",
+	},
+}
+
+func TestSummaryHappyPath(t *testing.T) {
+
+	mod := `acop`
+
+	for _, tc := range(testConfigs) {
+
+		commandAndArgs := []string{
+			"nonmem",
+			"summary",
+			filepath.Join(SUMMARY_TEST_DIR, mod, mod+".lst"),
+		}
+
+		if (tc.bbiFlag != "") {
```

Comment:
As a note, go does not require the parentheses

```go
 if tc.bbiFlag != "" {
}
```

is valid and more idiomatic